authors/list_latex.py

```diff
@@ -7,7 +7,11 @@
 author_list = load(open("authors.json"))
 
 with open("../authors.tex", "wb") as f:
```

Comment:
@asmeurer what's the underlying cause of this? Python 2 can sometimes be configured to use utf8 by default, and sometimes not, depending on a local user configuration? And so in order to be portable, one must always set this inside the `open` statement?